tests/codebuild/run_integtest.sh

```diff
@@ -88,22 +89,27 @@ pushd sagemaker-k8s-operator
     # Setup the PATH for smlogs
     mv smlogs-plugin/linux.amd64/kubectl-smlogs /usr/bin/kubectl-smlogs
 
-    # Goto directory that holds the CRD  
-    pushd sagemaker-k8s-operator-install-scripts
-        # Since OPERATOR_AWS_SECRET_ACCESS_KEY and OPERATOR_AWS_ACCESS_KEY_ID defined in build spec, we will not create new user
-        ./setup_awscreds
-
-        echo "Deploying the operator"
-        kustomize build config/default | kubectl apply -f -
-    popd 
-
+    # Allow for overriding the installation of the CRDs/controller image from the
+    # build scripts if we want to use our own installation
+    if [ "${SKIP_INSTALLATION}" == "true" ]; then
+        echo "Skipping installation of CRDs and operator"
+    else
+        # Goto directory that holds the CRD  
+        pushd sagemaker-k8s-operator-install-scripts
+            # Since OPERATOR_AWS_SECRET_ACCESS_KEY and OPERATOR_AWS_ACCESS_KEY_ID defined in build spec, we will not create new user
+            ./setup_awscreds
+
+            echo "Deploying the operator"
+            kustomize build config/default | kubectl apply -f -
+        popd
+
+        echo "Waiting for controller pod to be Ready"
+        # Wait to increase chance that pod is ready
+        # TODO: Should upgrade kubectl to version that supports `kubectl wait pods --all`
+        sleep 60
```

Comment:
You try can using `kubectl wait` instead of sleep though it is still experimental.
https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#wait

Reply:
We previously were.  We switched back to sleep because it was sometimes failing. @cadedaniel made this change back then.

Reply:
That's right. A bit more context:

We used to have `wait`, but there was a race condition because `wait --all` wasn't yet supported -- it required specification of a pod name. The race condition is that this line of code is executed before the pod is created by the deployment controller. Thus, we could still continue to use the `wait` code, but it was not very clean (something like `kubectl wait pod "$(code to get pod name)"`) _and_ required a `sleep` beforehand.. cleaner to just use `sleep 60`.

If `kubectl wait` has `--all` flag and is in good state, then no problem using in the future.